.gitignore

```diff
@@ -14,3 +14,6 @@ storybook-static
 coverage
 .storybook-out/
 lefthook-local.yml
+
+# IDE
+/.idea/
```

Comment:
@aleonov-virtru would you add this to your global git ignore and remove it from the project?

```
# Add a global gitignore file
git config --global core.excludesfile ~/.gitignore
# Add the Webstorm dir to the file
echo "/.idea/" >> ~/.gitignore
```